github.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+type prType struct {
+	client    *github.Client
+	ctx       *context.Context
+	owner     string
+	repo      string
+	OpenSince int
+	CloseOn   int
+	GHpr      *github.PullRequest
+}
+
+func actions(currentPr prType) {
+	var takeAction bool
+	var actionTaken action
+
+	for _, actionItem := range runConfig.Actions {
+		if currentPr.OpenSince >= actionItem.Day {
+			actionTaken = actionItem
+			takeAction = true
+			break
+		}
+	}
+	if takeAction {
```

Comment:
In order to avoid a extra `else` I'd tap this piece of code a bit, btw [see](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow)
```go

if !takeAction {
  fmt.Printf("[skipping] \n")
  return
}

if arguments["--enable"].(bool) {
  commentOnPr(currentPr, actionTaken.Message)
  switch actionTaken.Action {
  case "close":
    closePr(currentPr)
  default:
    fmt.Printf("[%s]\n", actionTaken.Action)
  }
} else {
  fmt.Printf("[%s 'DRY MODE']\n", actionTaken.Action)
}
```